packages/rum/src/domain/rumEventsCollection/view/trackViews.spec.ts

```diff
@@ -356,6 +356,46 @@ describe('rum track loading type', () => {
   })
 })
 
+describe('rum track view is active', () => {
+  let setupBuilder: TestSetupBuilder
+  let handler: jasmine.Spy
+  let getViewEvent: (index: number) => View
+
+  beforeEach(() => {
+    ;({ handler, getViewEvent } = spyOnViews())
+
+    setupBuilder = setup()
+      .withFakeClock()
+      .withFakeLocation('/foo')
+      .beforeBuild(({ location, lifeCycle }) => {
+        lifeCycle.subscribe(LifeCycleEventType.VIEW_UPDATED, handler)
+        return trackViews(location, lifeCycle)
+      })
+  })
+
+  afterEach(() => {
+    setupBuilder.cleanup()
+  })
+
+  it('should collect initial view as active', () => {
+    setupBuilder.build()
+    expect(getViewEvent(0).isActive).toBe(true)
+  })
+
+  it('should collect old view as inactive and new one as active after a route change', () => {
+    setupBuilder.build()
+    history.pushState({}, '', '/bar')
+    expect(getViewEvent(1).isActive).toBe(false)
+    expect(getViewEvent(2).isActive).toBe(true)
+  })
+
+  it('should collect view as active after a search change', () => {
+    setupBuilder.build()
+    history.pushState({}, '', '/foo?bar=qux')
+    expect(getViewEvent(1).isActive).toBe(true)
+  })
```

Comment:
what is interesting about search change?

Reply:
it's a case when the location changes but the view remains active. The goal was to make sure that the view doesn't become inactive after any location change.

Reply:
ok, can you clarify you intent in the test description then?